db/migrate/20170718111834_unify_container_definition_and_container.rb

```diff
@@ -40,37 +40,54 @@ def up
     add_column :containers, :capabilities_drop,  :string
     add_column :containers, :command,            :text
 
-    containers = Arel::Table.new(:containers)
-    definitions = Arel::Table.new(:container_definitions)
+    db_connection = ActiveRecord::Base.connection
+
     say_with_time("Copying over columns from container_definition to container") do
       %w(image image_pull_policy memory cpu_cores container_group_id privileged
          run_as_user run_as_non_root capabilities_add capabilities_drop command).each do |column|
-        join_sql = definitions.project(definitions[column.to_sym])
-                              .where(definitions[:id].eq(containers[:container_definition_id])).to_sql
-        Container.update_all("#{column} = (#{join_sql})")
+        subquery = <<-SQL
+          SELECT id, #{column} FROM container_definitions
+        SQL
+
+        db_connection.execute <<-SQL
+          UPDATE containers SET #{column} = subquery.#{column}
+          FROM (#{subquery}) AS subquery
```

Comment:
Would it make sense to just put `subquery` inline here?

I'm not sure how we typically use raw SQL.

I guess it does remove a little duplication - which ever you feel is best.

Personally, I find the sql you have written to be easy to read and much more readable than anything arel would give us.